src/main/java/org/truffleruby/core/VMPrimitiveNodes.java

```diff
@@ -573,26 +573,6 @@ protected Object shouldNotReachHere(Object message,
 
     }
 
-    @Primitive(name = "vm_java_version")
```

Comment:
This is useful for debugging (e.g., find out which JVM version was used to build a native image).
It should probably be a method on Truffe::Debug or Truffle::System for clarity, I'll do that.